microprofile/opentracing-smallrye/src/main/java/org/wildfly/extension/microprofile/opentracing/smallrye/TracingLogger.java

```diff
@@ -0,0 +1,73 @@
+package org.wildfly.extension.microprofile.opentracing.smallrye;
+
+import static org.jboss.logging.Logger.Level.DEBUG;
+import static org.jboss.logging.Logger.Level.INFO;
+import static org.jboss.logging.Logger.Level.WARN;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "WFLYTRAC", length = 4)
+public interface TracingLogger extends BasicLogger {
+  TracingLogger ROOT_LOGGER = Logger.getMessageLogger(TracingLogger.class, TracingLogger.class.getPackage().getName());
+
+  @LogMessage(level = INFO)
+  @Message(id = 1, value = "Activating MicroProfile OpenTracing Subsystem")
+  void activatingSubsystem();
+
+  @LogMessage(level = DEBUG)
+  @Message(id = 2, value = "MicroProfile OpenTracing Subsystem is processing deployment")
+  void processingDeployment();
```

Comment:
I see no need to remove these, but just as an FYI we only require INFO and higher to be internationalized.

Reply:
Those were INFO in an earlier iteration, but I downgraded them to DEBUG, as I found the log too verbose for the subsystem.